src/components/cms/SkillCreator/SkillWizard.js

```diff
@@ -679,6 +679,19 @@ class SkillWizard extends Component {
     });
   };
 
+  giveWarningMessage = (language, category, name) => {
+    let message = '';
+    if (language === '') {
+      message += 'language ';
+    }
+    if (category === null) {
+      message += 'category ';
+    }
+    if (name === '') {
+      message += 'name ';
+    }
+    return message;
```

Comment:
map through the array and insert `,` in between. 
And then return the entire message including `Please enter ...`.

Eg message - `Please do not leave the category and skill fields empty`.